### PR TITLE
Improving/naming conventions

### PR DESCRIPTION
**Pull request by ignaciocampo**
Made some Improvements in naming.

There was a Student Search component in the dashboard but the naming of it said Student Registration.
Changed it to Student Search because that is what is supposed to do.

Changed the name of MatchingCalendar component being displayed in the page,  previously it was Mentor Mentee Matching but it was too long and it wasn't displaying the whole name.
Changed the name to Matching Calendar since that is the component name and also you can see the whole name.

Changed the States and handleClicks naming in Matching Calendar components to make it more understandable.

### CODE REVIEW

**Review by amybarger — APPROVED** [review]
Your PR is very detailed and could understand everything you did and the reasoning. This looks good!

File: src/components/pages/Headmaster/MentorMenteeMatching/MatchingCalendar.js [hunk 1]
```diff
@@ -59,14 +58,14 @@ const MatchingCalendar = props => {
...
 
-  const [clicked, setClicked] = useState(false);
-  const [clicked2, setClicked2] = useState(false);
+  const [clickMenteeList, setClickMenteeList] = useState(false);
+  const [clickMentorList, setClickMentorList] = useState(false);
 
-  const handleClick = () => {
-    setClicked(!clicked);
+  const handleClickMenteeList = () => {
+    setClickMenteeList(!clickMenteeList);
```
Comment (HeyMichelle) [inline src/components/pages/Headmaster/MentorMenteeMatching/MatchingCalendar.js:65]: Yes these changes make more sense, thank you for the improvements

**Review by amybarger — APPROVED** [review]
Nice!

**Review by HeyMichelle — APPROVED** [review]
All of the changes are good to go, I like the naming conventions you used. Thank you!

**Review by amybarger — APPROVED** [review]

**Pull request merged**